controllers/storagecluster/cephblockpools.go

```diff
@@ -83,6 +86,29 @@ func (obj *ocsCephBlockPools) ensureCreated(r *StorageClusterReconciler, instanc
 				return err
 			}
 		}
+
+		if !reflect.DeepEqual(existing, cephv1.CephBlockPool{}) {
```

Comment:
Should this be done each time or only when the CephBlockPool is first created (switch case errors.IsNotFound)?